deb-get

```diff
@@ -2413,6 +2413,16 @@ function deb_mullvad-vpn() {
     SUMMARY="VPN Service based in Sweden that does not collect user data."
 }
 
+function deb_master-pdf-editor() {
+    if [ "${ACTION}" != "prettylist" ]; then
+        URL=$(curl -s -S "https://code-industry.net/get-master-pdf-editor-for-ubuntu/?download" | grep -Eo "href=\"https://code-industry.net/public/.*\.deb" | cut --complement -c -6)
```

Comment:
Multiple files are provided, depending on the version of Debian/Ubuntu. The correct file should be selected based on `UPSTREAM_CODENAME`, and the supported codenames should be listed in `CODENAMES_SUPPORTED`. Read [CONTRIBUTING#creating-the-function](https://github.com/wimpysworld/deb-get/blob/main/CONTRIBUTING.md#creating-the-function) for further information.

Reply:
There is one for Debian 8 - 10 / Ubuntu 14.x - 20.x and another for Debian 9 - 11 / Ubuntu 18.x - 22.x.

Reply:
one seems to include qt the other doesn't, there's one download button on the page
also 18.x - 22.x should cover everything from ubuntu to 18.x 22.x so that should include ubuntu 20.x while debian 9-11 should cover everything from debian 9 to 11, so that should include debian 10

deb-get doesn't support anything older than ubuntu anyway so the default download link should cover everything we need